src/components/Axis/Axis.jsx

```diff
@@ -108,6 +114,94 @@ const Axis = createClass({
 		const range = scale.range();
 		const sign = orient === 'top' || orient === 'left' ? -1 : 1;
 		const isH = orient === 'top' || orient === 'bottom'; // is horizontal
```

Comment:
Whooh this is a doozy

Reply:
Do you think it'd be _easier_ to supply a degree vs an enumarable value for textOrientation?

Reply:
Potentially. My only thinking is that the degrees we are using to rotate text here are not a clear 0, 45, and 90, instead it's 0, -30, and -90, which may make enumerable values the better option. Thoughts?

Reply:
I'm fine keeping it as enumerable values.